addons/base_setup/views/res_config_views.xml

```diff
@@ -9,147 +9,240 @@
                         <button string="Apply" type="object" name="execute" class="oe_highlight"/>
                         <button string="Cancel" type="object" name="cancel" class="oe_link"/>
                     </header>
-                    <separator string="General Settings"/>
-                    <div>
-                        <label string="You will find more options in your company details: address for the header and footer, overdue payments texts, etc."/>
-                        <button type="object" name="open_company" string="Configure your company data" icon="fa-cogs" class="oe_inline oe_link"/>
-                        <field name="company_id" invisible="1"/>
-                    </div>
-                    <group>
-                        <label for="id" string="Email" name="email_label"/>
-                        <div name="email">
-                            <div name="email.outgoing_server">
-                                <button type="action"
-                                    name="%(base.action_ir_mail_server_list)d"
-                                    string="Configure outgoing email servers" class="oe_link"/>
-                            </div>
-                        </div>
-                    </group>
-                    <group>
-                        <label for="id" string="Portal access"/>
-                        <div>
+                    <div class="o_form_sheet_bg">
+                        <div class="o_form_sheet">
+                            <separator string="General Settings"/>
                             <div>
-                                <field name="module_portal" class="oe_inline"/>
-                                <label for="module_portal"/>
-                                <p>
-                                    When you send a document to a customer
-                                    (quotation, invoice), your customer will be
-                                    able to signup to get all his documents,
-                                    read your company news, check his projects,
-                                    etc.
-                                </p>
+                                <h2>Company</h2>
+                                <div class="row mt16 o_settings_container">
+                                    <div class="col-md-6 o_setting_box">
+                                        <div class="input-group">
+                                            <div class="o_setting_left_pane"/>
+                                            <div class="o_setting_right_pane">
+                                                <label string="Company Details"/><i class="fa fa-building" aria-hidden="true"></i>
```

Comment:
If user has multi companies access right, then only show this icon on next to box title at top right with following tooltip: Values set here are company-specific.